src/Modules/Frontend/templates/base/Head.html.twig

```diff
@@ -3,8 +3,8 @@
 <head>
   <meta charset="utf-8" />
   <meta name="generator" content="Fork CMS" />
-{#  {{ meta|raw }}#}
-{#  {{ metaCustom|raw }}#}
+{# {{ meta|raw }} #}
```

Comment:
@carakas waarom staan die in commentaar?

Reply:
Not sure, might be that I haven't gotten that to work yet, I'll have to investigate